src/utils/sanitizePackageName.js

```diff
@@ -0,0 +1,5 @@
+const unsafeRegex = /[/*]/g;
```

Comment:
Let's go ahead and replace _all_ invalid filename characters to avoid future issues:

`/, \, <, >, :, ", |, ?, *`

On windows, a filename also cannot end in `.`

Reply:
okay. if we want to create a more generic sanitizeFileName then we should use ‘sanitize-filename’ itself from npm, i think. as there are a bunch more illegal windows names.

i’ll do this in a couple hours time when i get home

Reply:
i prefer the the approach of trusting the package name to be a valid package name. I understand the want for a more generic approach, in case at some point you'd want to use this for things other than the package name.
But, if that was to happen, it would take no more work to include the login then that it would now, and trusting the package name isn't locking us in to anything that is complex or would mean a lot of rework should a future decision mean it needs changed.
But, i'm also fine with it and have done it and it's done now.